book/installation.rst

```diff
@@ -269,6 +269,14 @@ If there are any issues, correct them now before moving on.
 
         setfacl isn't available on NFS mount points. However, setting cache
         and logs over NFS is strongly not recommended for performance.
+        
+    .. note::
+
+        When not using PHP's built-in web server, make sure that the web server
+        process runs with your user account or to configure advanced rules (like ACLs).
+        
+        http://symfony.com/doc/current/cookbook/configuration/web_server_configuration.html.
+        
```

Comment:
I agree with Ryan here. It looks like a duplication to me. Maybe I'm missing something here?